packages/react-ui-core/src/List/List.js

```diff
@@ -70,7 +70,13 @@ export default class List extends PureComponent {
         )}
       >
         {items.map((item, i) => (
-          <Item key={this.itemId(i)} nodeType={listItemNodeType} {...props}>
+          <Item
+            key={this.itemId(i)}
+            index={i}
+            nodeType={listItemNodeType}
+            theme={theme}
```

Comment:
Why are you passing theme here?

Reply:
Not sure. Will fix.

Reply:
It isn't present in the spread `props` because it was destructed, and we needed access to it in 'SearchBar/ItemWrapper'

```
render() {
    const {
      selectedIndex,
      index,
      theme,
      children,
      onSelectionClick,
    } = this.props

    return (
      <div
        className={cn(
          theme.ListItem,
          selectedIndex === index && theme.Selected
        )}
        onClick={onSelectionClick}
        onMouseEnter={this.handleHover}
      >
        {children}
      </div>
    )
  }
```